pkgs/development/libraries/libuv/default.nix

```
@@ -22,14 +22,14 @@
 , python3
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "1.45.0";
   pname = "libuv";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "v${version}";
+    owner = finalAttrs.pname;
+    repo = finalAttrs.pname;
```

overriding `pname` should not change the source repo

```suggestion
    owner = "libuv";
    repo = "libuv";
```